src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+import core.basesyntax.figures.Figure;
+import core.basesyntax.supplier.FigureSupplier;
+
+public class Main {
+    public static void main(String[] args) {
+        final int Array_Size = 5;
```

Comment:
![image](https://github.com/mate-academy/jv-oop-advanced/assets/20326858/f2931f35-17a4-45fb-b6ef-5d24318aeee6)

Reply:
do i need to make an array size variable as CONSTANTS ?

Reply:
@Rommelua do i need to make an array size variable as CONSTANTS ?